src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -358,6 +451,9 @@ public int SizeOfSet
             }
         }
 
+        /// <summary>
+        /// whether the element is resizeable
```

Comment:
"resizeable" is an accepted alternative spelling, but "resizable" (with no 'e' after the z) is the more common spelling

```suggestion
        /// whether the element is resizable
```